tck/src/main/java/org/eclipse/microprofile/config/tck/CDIPlainInjectionTest.java

```diff
@@ -51,11 +51,15 @@
  */
 public class CDIPlainInjectionTest extends Arquillian {
 
+    private static final String DEFAULT_PROPERTY_BEAN_KEY = "org.eclipse.microprofile.config.tck.CDIPlainInjectionTest.defaultPropertyBean.configProperty";
```

Comment:
No, because the javadoc mentions the first letter is [decapitalized](https://github.com/eclipse/microprofile-config/blob/3abbaa9cf8351ea4da49b64a3188e32c91be6790/api/src/main/java/org/eclipse/microprofile/config/inject/ConfigProperty.java#L105) (as Steve mentioned as well in his report)

Also, since this is only mentioned in javadocs I think I'm going to update the actual spec to reflect this.